doc/design.md

```diff
@@ -127,13 +127,13 @@ The solution is to have the bridge wait until a publisher becomes available befo
 
 Due to possible delays induced by the bridge, it is possible that the original deadline of a publisher may not be met when forwarding to a different domain.
 Furthermore, if deadline is important for the user application, then it is probably better for them to set manually on the bridge.
```

Comment:
I'm not following this sentence:

".. if deadline is important for the user application, then it is probably better for them to set manually on the bridge."

Is "them" referring to the deadline? If so, it seems to me it should be either:

* "... if deadlines are important for the user application, then they should be set manually on the bridge.", or
* "... if a deadline is important for the user application, then it should be set manually on the bridge."

Reply:
Updated in f5dcaf8
Let me know if it reads better now.

Reply:
Yes, clear now, thanks.